pages/settings/admin/apikeys.vue

```diff
@@ -49,8 +58,13 @@ export default Vue.extend({
   data() {
     return {
       apiKeys: [] as AuthenticationInfo[],
-      addingNewKey: false,
-      newKeyAppName: ''
+      newKeyAppName: '',
+      loading: false
```

Comment:
This uses async data, so I don't think loading on the buttons is necessary.

Reply:
Loading is shown while the request (add/remove) api is being processed, which can be seen in slow networks.

Reply:
Ah ok, we should probably specify which area is loading, so a loading state for addApiKeys, removeApiKeys...